.travis.yml

```diff
@@ -3,6 +3,7 @@ cache: bundle
 sudo: false
 rvm:
   - 2.3.3
```

Comment:
Can you remove this? I'm confident we're backwards compatible to this version when `2.4.1` is green.